amundsen_application/static/js/components/DashboardPage/ImagePreview/index.tsx

```diff
@@ -43,7 +43,7 @@ const PreviewModal = ({ imageSrc, onClose }: PreviewModalProps) => {
         </Modal.Title>
       </Modal.Header>
       <Modal.Body>
-        <img src={imageSrc} height="auto" width="100%" />
+        <img src={imageSrc} height="auto" width="100%" alt="" />
```

Comment:
Great question! Here is from w3.org:

"For example, the information provided by the image might already be given using adjacent text, or the image might be included to make the website more visually attractive. In these cases, a null (empty) alt text should be provided ( alt="" ) so that they can be ignored by assistive technologies, such as screen readers."

Reply:
Yes, so in this case are you leaving it empty because the modal title says `Dashboard Preview`, therefore giving context about this image?

Reply:
Well, in reality I was putting the empty alt to just be compliant. I don't think a visual impaired user would even open the modal, as it is a preview image of the dashboard.

Reply:
I see, I'll look at each of these on a case by case basis to see if there are any where we probably should provide some text.